elementary/monitor/api/tests/tests.py

```diff
@@ -149,18 +140,19 @@ def get_test_info_from_test_metadata(
         if metadata.test_type == "dbt_test":
             configuration = dict(
                 test_name=metadata.test_name,
-                test_params=try_load_json(metadata.test_params),
+                test_params=metadata.test_params,
             )
         else:
             configuration = dict(
                 test_name=metadata.test_name,
-                timestamp_column=test_params.get("timestamp_column"),
-                testing_timeframe=test_params.get("timeframe"),
-                anomaly_threshold=test_params.get("sensitivity"),
+                timestamp_column=metadata.test_params.get("timestamp_column"),
```

Comment:
Are you sure metadata.test_params exists / not null?

Reply:
Yeah because by default we set it to `{}` if its empty (on the pydantic schema)